lib/chef/knife/vsphere_vm_clone.rb

```diff
@@ -368,15 +368,17 @@ def run
   def wait_for_access(connect_host, connect_port, protocol)
     if protocol == 'winrm'
       load_winrm_deps
+      if get_config(:winrm_transport) == 'ssl' && get_config(:winrm_port) == '5985'
+        config[:winrm_port] = '5986'
```

Comment:
Thank for the patch. At first glance [this](https://github.com/chef/knife-windows/blob/master/lib/chef/knife/winrm_base.rb#L59) should be doing the work for you, is that not the case?